tools/clang/test/SemaHLSL/hlsl/types/invalid_longvec_decls.hlsl

```diff
@@ -0,0 +1,132 @@
+// RUN: %dxc  -DTYPE=float -DNUM=7 -T ps_6_9 -verify %s
```

Comment:
Reminder to self to add tests for vectors longer than 1024 as well as structs with base classes containing long vectors in all the forbidden locations.

Reply:
Add in toolong_vectors.hlsl